src/PointwiseFunctions/AnalyticData/GrMhd/SpecInitialData.cpp

```diff
@@ -100,47 +100,49 @@ template <size_t ThermodynamicDim>
 template <typename DataType>
 void SpecInitialData<ThermodynamicDim>::VariablesComputer<DataType>::operator()(
     const gsl::not_null<Scalar<DataType>*> specific_internal_energy,
-    const gsl::not_null<Cache*> /*cache*/,
+    const gsl::not_null<Cache*> cache,
     hydro::Tags::SpecificInternalEnergy<DataType> /*meta*/) const {
   const auto& rest_mass_density =
       get<hydro::Tags::RestMassDensity<DataType>>(interpolated_data);
+  const auto& temperature =
+      cache->get_var(*this, hydro::Tags::Temperature<DataType>{});
   const size_t num_points = get_size(get(rest_mass_density));
   for (size_t i = 0; i < num_points; ++i) {
     const double local_rest_mass_density =
         get_element(get(rest_mass_density), i);
-    if (local_rest_mass_density <= density_cutoff) {
-      get_element(get(*specific_internal_energy), i) = 0.;
-    } else {
+    const double local_temperature =
+        get_element(get(temperature), i);
       if constexpr (ThermodynamicDim == 1) {
```

Comment:
Can you format from line 113->127? I think these don't need to be indent as much